pages.de/common/dotnet.md

```diff
@@ -0,0 +1,20 @@
+# dotnet
+
+> Plattformübergreifende Kommandozeilenandwendungen für .NET Core.
+> Mehr Informationen: <https://docs.microsoft.com/de-de/dotnet/core/tools/>.
```

Comment:
We don't use i18n in MSFT URLs because he choose your language automatically. Should we do the same in non-English pages? cc @owenvoke @schneiderl 
```suggestion
> Mehr Informationen: <https://docs.microsoft.com/dotnet/core/tools/>.
```

Reply:
Yeah, that definitely seems like a good idea.

Reply:
Yeah, I think it's best across all pages with MS links.